payslip/src/Main.java

```diff
@@ -0,0 +1,9 @@
+public class Main {
+    public static void main(String[] args) {
+        Payslip payslip = new PayslipBuilder().setHra(10).setDa(75).getPaySlip();
```

Comment:
What is the problem if I directly use Payslip constructor? still the same right.?
Payslip p = new Payslip(10,75,0);
p.calculateNetSalary(20000);